install/kubernetes/helm/istio/charts/grafana/values.yaml

```diff
@@ -4,7 +4,7 @@
 enabled: false
 replicaCount: 1
 image:
-  repository: grafana/grafana
+  hub: docker.io/grafana
```

Comment:
I don't think this is correct without an image name.  @ibm-jason can you ping @sdake to get his comment addressed so we can get this merged for master and possibly 1.2?

Reply:
@linsun sure, np.

Reply:
Prometheus, Grafana and Kiali are third-party add-on components. 
 We use the direct upstream images as they are maintained more effectively than we could as a project.  Please follow the pattern in Prometheus:

https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/prometheus/values.yaml#L6

Remove `image:`
make hub and tag top level K/Vs in the grafana chart.
Keep the Hub as you have it.

The full rendered `image` field for the grafana chart should look like `docker.io/grafana/grafana:6.1.6`

To make that happen, use the pattern again from Prometheus:
https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/prometheus/templates/deployment.yaml#L33

Cheers
-steve

Reply:
I agree with everything you said, but if we are going to to change the values maybe it should be done in https://github.com/istio/installer. Otherwise this is backwards incompatible